xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -1521,3 +1521,20 @@ HDR_STATUS CWIN32Util::GetWindowsHDRStatus()
 
   return status;
 }
+
+void CWIN32Util::PlatformSyslog()
+{
+  CLog::Log(LOGINFO, "System has {:.1f} GB of RAM installed",
+            GetSystemMemorySize() / static_cast<double>(MB));
+  CLog::Log(LOGINFO, "%s", GetResInfoString().c_str());
+  CLog::Log(LOGINFO, "Running with %s rights",
+            (IsCurrentUserLocalAdministrator() == TRUE) ? "administrator"
+                                                        : "restricted");
+  CLog::Log(LOGINFO, "Aero is %s", (g_sysinfo.IsAeroDisabled() == true) ? "disabled" : "enabled");
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)